pages/[id]/results/[field].vue

```diff
@@ -11,34 +11,15 @@
           </tr>
         </thead>
         <tbody>
-          <tr v-for="team in data" :key="team.knrbid" class="hover">
-            <td>
-              <BladeField :code="team.clubnameshort" />
-            </td>
-            <td>
-              <TeamField
-                :team-name="team.teamname"
-                :back-number="team.times[0].backnumber"
-                :rower8="team.rower8 ?? ''"
-              />
-            </td>
-            <template v-for="time in team.times">
-              <td v-for="subTime in removeDuplicates(time.times)" :key="subTime.name">
-                <TimeField :sub-time="subTime" :is-altered="true" />
-              </td>
-            </template>
-          </tr>
+          <TeamRowResult v-for="team in data" :key="team.knrbid" :team="team"></TeamRowResult>
```

Comment:
Elements zonder content kun je gelijk sluiten:
```suggestion
          <TeamRowResult v-for="team in data" :key="team.knrbid" :team="team" />
```